[2023-04-14 17:00:01,645] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T16:59:59.262607+00:00 [queued]>
[2023-04-14 17:00:01,659] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T16:59:59.262607+00:00 [queued]>
[2023-04-14 17:00:01,662] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:00:01,664] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 17:00:01,665] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:00:01,675] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-14 16:59:59.262607+00:00
[2023-04-14 17:00:01,687] {standard_task_runner.py:52} INFO - Started process 97 to run task
[2023-04-14 17:00:01,694] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'manual__2023-04-14T16:59:59.262607+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmp696l_5xt', '--error-file', '/tmp/tmp0vtndonz']
[2023-04-14 17:00:01,697] {standard_task_runner.py:80} INFO - Job 69: Subtask get_cv_lac_data
[2023-04-14 17:00:01,812] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T16:59:59.262607+00:00 [running]> on host d014c8b814c7
[2023-04-14 17:00:01,914] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T16:59:59.262607+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T16:59:59.262607+00:00
[2023-04-14 17:00:01,919] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-14 17:00:01,922] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 17:00:01,945] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T165959, start_date=20230414T170001, end_date=20230414T170001
[2023-04-14 17:00:01,999] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 17:00:02,062] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
